packages/cli/src/credentials/credentials.controller.ee.ts

```diff
@@ -53,16 +49,12 @@ EECredentialsController.get(
  * GET /credentials/:id
  */
 EECredentialsController.get(
-	'/:id',
+	'/:id(\\d+)',
 	(req, res, next) => (req.params.id === 'new' ? next('router') : next()), // skip ee router and use free one for naming
 	ResponseHelper.send(async (req: CredentialRequest.Get) => {
 		const { id: credentialId } = req.params;
 		const includeDecryptedData = req.query.includeData === 'true';
 
-		if (Number.isNaN(Number(credentialId))) {
-			throw new ResponseHelper.BadRequestError(`Credential ID must be a number.`);
-		}
```

Comment:
Still useful to validate the request? Same elsewhere.

Reply:
I don't think we need to. those urls become 404 instead, which makes a lot more sense to me than 400.